src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -270,7 +325,43 @@ export const NewProposal = ({ navigation, data, query }) => {
   );
 };
 
+const ImageSelector = ({ control, imageUri, item, onChange, selectImage, value }) => {
+  const { buttonTitle, infoText, label, name } = item;
+
+  useEffect(() => {
+    // the `!value` control is used for editing the proposal
+    // the proposal contains an image, it is for showing the preview to the users
+    if (!value) {
+      onChange(imageUri);
+    }
+  }, [imageUri]);
+
+  return (
+    <>
+      <Input hidden label={label} control={control} name={name} value={value} />
+      <RegularText smallest placeholder>
+        {infoText}
+      </RegularText>
+
+      {value ? (
+        <WrapperRow center spaceBetween>
+          <Image source={{ uri: value }} style={styles.image} />
+          <Touchable onPress={() => onChange(null)}>
```

Comment:
would a `reset({ image: '' })` do the same here, which would be more readable than some `onChange(null)` without context.

Reply:
ok, let's keep it

```
onChange('');
setErrorMessage('');
```

this seams clear to me